server/src/test/java/org/elasticsearch/index/mapper/DateFieldTypeTests.java

```diff
@@ -318,7 +318,10 @@ public void testRangeQuery() throws IOException {
             LongPoint.newRangeQuery("field", instant1, instant2),
             SortedNumericDocValuesField.newSlowRangeQuery("field", instant1, instant2)
         );
+        // TODO: temporarily disable - the range query rewrites to a MatchNoDocsQuery
```

Comment:
I'm going to follow up on this.

Reply:
Interesting, shout if you need help on this! this is probably due to https://github.com/apache/lucene/pull/14609/ ?

Reply:
I guess that this case can no longer be tested the way that we do, because we have no data in the index have we?